openlibrary/plugins/openlibrary/js/automatic.js

```diff
@@ -11,11 +11,13 @@ export default function($) {
     });
 
     // wmd editor
-    $('textarea.markdown').wmd({
-        helpLink: '/help/markdown',
-        helpHoverTitle: 'Formatting Help',
-        helpTarget: '_new'
-    });
+    $markdownTextArea = $('textarea.markdown');
```

Comment:
Could we move the import to the file where the other dynamic imports are? It would be kind of nice to keep them all in one place. (that's also the file with the promise polyfill, but I'm not sure if that's a problem).

Reply:
done